packages/frontend/src/queries/uniswap/positionsQuery.ts

```diff
@@ -77,4 +77,43 @@ export const POSITIONS_QUERY = gql`
     }
   }
 `
+
+export const ACTIVE_POSITIONS_QUERY = gql`
+  query activePositions($poolAddress: String!, $owner: String!) {
+    positions(where: { pool: $poolAddress, owner: $owner, liquidity_gt: 0 }) {
```

Comment:
Have a q, do we need to put orderBy timeStamp in this query?

Reply:
Nope not needed for now. If needed can add later